### PR TITLE
Fix(2 piscine-go exercise solutions)

### PR DESCRIPTION
**Pull request by MSilva95**
Fix printStr solution upgrade
rotatevowels solution fix accordinf to dav review

### CODE REVIEW

**Review by davhojt — APPROVED** [review]
Let's not mix unrelated things into a single PR.

For example, if we were going to conduct a wholesale switch of integer literals to rune literals, then it would be reasonable to include many files in a single PR.

But changing `fmt.PrintLn` to `fmt.Print` doesn't appear related.

File: solutions/printstr.go [hunk 1]
```diff
@@ -3,5 +3,5 @@ package solutions
 import "fmt"
 
 func PrintStr(s string) {
-	fmt.Print(s)
+	fmt.Println(s)
```
Comment (davhojt) [inline solutions/printstr.go:6]: The exercise this relates to should not be new-line terminated.

**Review by davhojt — CHANGES_REQUESTED** [review]
Sorry, I read it back to front before I approved before.

File: solutions/printstr.go (hunk 1 above)
Comment (MSilva95) [reply solutions/printstr.go:6]: Yes, but in my opinion we should have it with the new-line at the end, i added a change in the pr on public, but if you think we should not change it let me know and i will remove it.
<https://github.com/01-edu/public/pull/1378/files>

Comment (rcatini) [reply solutions/printstr.go:6]: @MSilva95 With the [upcoming new prompt](https://github.com/01-edu/public/pull/1357#discussion_r936788404) the example in the subject should become much clearer to understand (but I'm not sure when the new prompt will be introduced).

Comment (MSilva95) [reply solutions/printstr.go:6]: yes but im not sure for when is the new prompt so i changed the exercise, but maybe i will let it stay like it was and i will speak with Dav to start implementing the new prompt

**Comment by davhojt** [comment]
Exceptionally, I will merge this to master with a merge commit. The branch has been previously force-pushed so some references have changed.

**Pull request merged**